packages/server-wallet/src/engine/store.ts

```diff
@@ -357,6 +357,11 @@ export class Store {
     const results = await ObjectiveModel.query(this.knex).where({status: 'approved'});
     return results.map(o => o.toObjective());
   }
+  public async updateObjectiveProgressDate(objectiveIds: string[]): Promise<WalletObjective[]> {
```

Comment:
Is this method ever used?

Reply:
[Yup](https://github.com/statechannels/statechannels/blob/eventless-engine/packages/server-wallet/src/wallet/wallet.ts#L254:L254).

Reply:
I thought my vs-code editor failed to find it. It must have been either user-error or a glitch.